website/pages/docs/reference/types.md

```diff
@@ -0,0 +1,10 @@
+# CloudQuery Types
+
+CloudQuery uses [Apache Arrow](https://arrow.apache.org/docs/index.html) to represent data internally. Source plugins define columns in terms of Arrow types, and destinations support converting from Arrow to their own native types.
```

Comment:
```suggestion
CloudQuery uses [Apache Arrow](https://arrow.apache.org/docs/index.html) to represent data internally. Source plugins define columns in terms of Apache Arrow types, and destinations support converting from Apache Arrow to their own native types.
```

Reply:
We don't need to include `Apache` every time we reference Arrow, it is implied after the first time. Even the Arrow project itself doesn't say "Apache Arrow" every single time (e.g. here https://arrow.apache.org/docs/format/Columnar.html)